event/sampler_chain.go

```diff
@@ -0,0 +1,37 @@
+package event
+
+import "github.com/DataDog/datadog-trace-agent/model"
+
+// samplerChain is an APMEvent sampler that submits an event to a sequence of samplers, returning as soon as one of
+// the samplers makes a decision.
+type samplerChain struct {
+	samplers []Sampler
+	callback func(decision SamplingDecision)
+}
+
+// NewSamplerChain creates a new sampler chain from the provided samplers and with the specified decision callback.
+func NewSamplerChain(samplers []Sampler, callback func(decision SamplingDecision)) Sampler {
+	return &samplerChain{
+		samplers: samplers,
+		callback: callback,
+	}
+}
+
+// Sample returns the first !NO_DECISION sampling decision from calls to the underlying samplers, in order.
```

Comment:
```suggestion
// Sample returns the first DecisionDontSample sampling decision from calls to the underlying samplers, in order.
```
What does it mean? Does it always return this value? If none of the samplers return this value what happens then? We should try to clarify a bit.